### PR TITLE
[styles] Fix parsing tag name with _ symbol

### PR DESCRIPTION
**Pull request by kshalnev**

### CODE REVIEW

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

**Comment by Zverik** [comment]
LGTM

File: indexer/indexer_tests/drules_selector_parser_test.cpp [hunk 1]
```diff
@@ -16,6 +16,16 @@ UNIT_TEST(TestDruleSelectorIsSet)
   TEST_EQUAL(SelectorOperatorIsSet, e.m_operator, ());
 }
 
+UNIT_TEST(TestDruleSelectorIsSet2)
+{
+  SelectorExpression e;
+  TEST(ParseSelector("envelope_area", e), ());
```
Comment (deathbaba) [inline indexer/indexer_tests/drules_selector_parser_test.cpp]: I suggest to use limit_rect - envelope_area is a complete mystery and hard to understand.

Comment (kshalnev) [inline indexer/indexer_tests/drules_selector_parser_test.cpp]: limit_rect does not reflect that it is area (sq.m.), besides, postgis use the same term, so, it is not so mistery :)

Comment (kshalnev) [inline indexer/indexer_tests/drules_selector_parser_test.cpp]: discussed: bbox_area

**Comment by deathbaba** [comment]
LGTM after changing to bbox_area

**Comment by kshalnev** [comment]
@Zverik, please merge

**Pull request merged**